<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kein Bild geladen...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>horizontal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="vCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>vertikal</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtergröße:</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Skalierung:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="scaleSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Verschiebung:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="transformSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
    <property name="value">
     <number>127</number>
    </property>
   </widget>
   <widget class="QLabel" name="imageViewLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>541</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
